src/Flicking.ts

```diff
@@ -677,8 +697,7 @@ class Flicking extends Component {
       maintained.forEach(([, next]) => { viewport.updateCheckedIndexes({ min: next, max: next }); });
     }
     panelManager.replacePanels(newPanels, newClones);
-
-    !isCallSync && this.viewport.beforeSync();
```

Comment:
How `vieweport.beforeSync` can be removed? What is the replacement of `viewerport.beforeSync`?

Reply:
That causes the case I've told you that call count of render() increases.
Because `viewport.beforeSync` calls updateVisiblePanels, and as it can trigger the `visibleChange` event, it can increase rendering cost about twice as much.
I've handled that by using `this.flag` (I'll change its name soon).